src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -4,6 +4,7 @@
   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
   xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
   xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+  xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"  
```

Comment:
yeah this is standard xaml convention - if we refer to a namespace in a different assembly, this is how you do it.

---
In reply to: [132083672](https://github.com/NuGet/NuGet.Client/pull/1623#discussion_r132083672) [](ancestors = 132083672)